editors/webstorm/README.md

```diff
@@ -43,3 +43,16 @@ Go to *File | Settings | Tools | File Watchers* for Windows and Linux or *WebSto
 * **Immediate file synchronization**: Uncheck to reformat on Save only (otherwise code will jump around while you type).
 
 ![Example](./prettier-file-wacther.png)
+
+### Multiple file types  ###
+
+If you woud like to use one watcher for multiple file types.
+Or if you like more control, make these changes to the settings above.
+
+* **File Type**: Any
+* **Arguments** set `--write $FileDir$/*{.js,.ts,.tsx}`
```

Comment:
I don't use webstorm, but I think this will format all the (js/ts/tsx) files in the current directory, not just the one you're invoking prettier on. I don't think this is what you want?

Reply:
Probably something like `--write $FilePathRelativeToProjectRoot$`